### PR TITLE
Body

### PR DESCRIPTION
**Pull request by palmerelias**
A pull request is ready to review.

### CODE REVIEW

**Comment by palmerelias** [comment]
Fixed the grammar issue for now. Will add bigger changes later.

File: src/App.jsx [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-        <img />
+        <DarkModeOutlinedIcon className='dark-mode' fontSize='large'/>
       </div>
       <div>
         <h1 className='ocupation'>Frontend</h1>
         <h1 className='ocupation'>Developer</h1>
         <h4 className='ocupation-message'>I like to craft solid and scalable frontend projects with great user experiences.</h4>
-        <h4 className='ocupation-message'>My names is Elias Palmer</h4>
+        <h4 className='ocupation-message-orange'>My name's Elias Palmer</h4>
       </div>
```
Comment (akselipalmer) [inline src/App.jsx:18]: looks like you installed MUI but aren't using the built in components.

File: src/App.jsx [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
-      <div>
+      <div className='bar'>
         <h3 className='name'>Elias</h3>
-        <img />
+        <DarkModeOutlinedIcon className='dark-mode' fontSize='large'/>
       </div>
       <div>
         <h1 className='ocupation'>Frontend</h1>
         <h1 className='ocupation'>Developer</h1>
         <h4 className='ocupation-message'>I like to craft solid and scalable frontend projects with great user experiences.</h4>
```
Comment (akselipalmer) [inline src/App.jsx:16]:
```suggestion
      <Box>
        <Typography variant="h1" className='ocupation'>Frontend</Typography>
        <Typography variant="h1"  className='ocupation'>Developer</Typography>
        <Typography variant="h4"  className='ocupation-message'>I like to craft solid and scalable frontend projects with great user experiences.</Typography >
```

MUI makes it easy to add divs and h1s by using their built in components. 
https://mui.com/material-ui/react-typography/
https://mui.com/material-ui/react-box/

You don't need to use these. But I would recommend giving them a try and see if you can get it to work. Mae sure you import the components. You can find tons of examples in the documentation I linked above

**Review by akselipalmer — APPROVED** [review]
Checkout my comments that I left in the code for ways to make it even better. Looks great! 😜👌✔

**Pull request merged**